### PR TITLE
PPTP-1219 - Input business contact - Phone number

### PR DESCRIPTION
**Pull request by kiranprakashdigital**
### Description of Work carried through

Brief description of what/why/how.

#### Check list 
 - [X] `./precheck` was executed (Integration/Component/Unit tests)
 - [X] `sbt scalafmt test:scalafmt` was executed
 - [X] User Acceptance Tests (UAT) were run locally and have passed.
 - [X] No Accessibility errors/warnings reported by Wave
 
Screenshot_20211005_133901

### CODE REVIEW

**Review by tmc-ee — APPROVED** [review]

File: app/uk/gov/hmrc/plasticpackagingtax/registration/views/phone_number_page.scala.html [hunk 1]
```diff
@@ -15,35 +15,35 @@
...
-@govukLayout(
-  title = Title("primaryContactDetails.phoneNumber.title"),
-  backButton = Some(BackButton(messages("site.back"), pptRoutes.ContactDetailsEmailAddressController.displayPage(), messages("site.back.hiddenText")))) {
-  @errorSummary(form.errors)
 
-  @sectionHeader(messages("primaryContactDetails.sectionHeader"))
-
-  @formHelper(action = pptRoutes.ContactDetailsTelephoneNumberController.submit(), 'autoComplete -> "off") {
+@phoneNumberField = @{form("value")}
+@contactName = @{request.registration.primaryContactDetails.name.getOrElse("")}
```
Comment (RyanKellyHMRC) [inline app/uk/gov/hmrc/plasticpackagingtax/registration/views/phone_number_page.scala.html:43]: You could look up a default string to use if no name is present. `primaryContactDetails.fullName.default` has been added to `messages.en` for this purpose. It should probably be changed to "the main contact".

File: conf/messages.en [hunk 2]
```diff
@@ -147,8 +147,10 @@ primaryContactDetails.tooManyAttempts.passcode.title = You have entered an incor
 primaryContactDetails.tooManyAttempts.passcode.1 = You have entered an incorrect passcode too many times
 primaryContactDetails.tooManyAttempts.passcode.2 = You must log out of you Government Gateway account and log back in to restart your registration.
 
-primaryContactDetails.phoneNumber.title = What is the primary contact’s telephone number?
-primaryContactDetails.phoneNumber.hint = This must be 11 numbers. For example 01234 567890.
+primaryContactDetails.phoneNumber.default = the primary contact
```
Comment (RyanKellyHMRC) [inline conf/messages.en:150]: I think we ought to have a single definition of the default "main contact".

File: tampermonkey/PPT_AutoComplete.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 // ==UserScript==
 // @name         PPT Registration AutoComplete
 // @namespace    http://tampermonkey.net/
-// @version      14.5
+// @version      14.6
```
Comment (RyanKellyHMRC) [inline tampermonkey/PPT_AutoComplete.js:4]: Nice 😁

**Review by RyanKellyHMRC — CHANGES_REQUESTED** [review]
Couple of minor changes requested. Thanks.

File: app/uk/gov/hmrc/plasticpackagingtax/registration/views/phone_number_page.scala.html (hunk 1 above)
Comment (kiranprakashdigital) [reply app/uk/gov/hmrc/plasticpackagingtax/registration/views/phone_number_page.scala.html]: Fixed

File: conf/messages.en (hunk 2 above)
Comment (kiranprakashdigital) [reply conf/messages.en]: Removed

**Pull request merged**